src/worker.cc

```diff
@@ -58,6 +58,7 @@ const std::unordered_map<unsigned, valhalla::valhalla_exception_t> error_codes{
     {112, {112, "Insufficiently specified required parameter 'locations' or 'sources & targets'", 400, HTTP_400, OSRM_INVALID_OPTIONS, "matrix_locations_parse_failed"}},
     {113, {113, "Insufficiently specified required parameter 'contours'", 400, HTTP_400, OSRM_INVALID_OPTIONS, "contours_parse_failed"}},
     {114, {114, "Insufficiently specified required parameter 'shape' or 'encoded_polyline'", 400, HTTP_400, OSRM_INVALID_OPTIONS, "shape_parse_failed"}},
+    {115, {115, "Insufficiently specified required parameter 'action'", 400, HTTP_400, OSRM_INVALID_OPTIONS, "action_parse_failed"}},
```

Comment:
what did you think of this refactor? i hoped by putting all this crap together it would be much easier to do all the things you needed for a new service level error

Reply:
if you're asking how I like the "new" way of defining valhalla_exceptions: love it:) I guess your comment is unrelated to the actual change?